### ISSUE TITLE
Add variables for document, window, xaml, and make dotting into them work

### ISSUE THREAD

**Issue by jschementi** [jschementi/iron#14]

**Comment by jschementi** [comment]
Should this just be a per-language implementation, rather than hoping it works correctly for all languages?

**Comment by jschementi** [comment]
Currently have document, window, and me (instead of xaml, though "me" is a bad name), and dotting-into works, but I have a messed up IDO implementation which need tending to and testing.

**Comment by jschementi** [comment]
Should this be some module to include?

```
# ruby
include Microsoft::Scripting::Silverlight::Helpers

# python
from Microsoft.Scripting.Silverlight.Helpers import *
```

**Comment by jschementi** [comment]
Also need to figure out the best solution for "dotting-into", be it IDO, or some language-specific method

**Comment by jschementi** [comment]
I'm just sticking to a language-specific solution for dotting-into. Also, the helper methods will be defined in a language-specific way as well, since for Ruby it makes to define them on Kernel, while Python would prefer them as top-level methods.

**Comment by jschementi** [comment]
Fixed in 7b30f07f819ee063c2b7a0761a45fa646e4c5048

Ruby now defines all these helpers and provides a simpler API for getting elements, getting/setting properties and attributes, setting events. etc. Python still uses the extension types. The helpers are auto-requires/imported.
